### PR TITLE
Address duplicates and outliers

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

File: R/createFigure4.R [hunk 1]
```diff
@@ -222,17 +221,8 @@ secchi.filtered <- secchi.data %>%
...
-unitConversion <- data.frame(units = c("m","ft","cm"),
-                             conversion = c(0.3048,1,30.48),
-                             stringsAsFactors = FALSE)
-
-secchi.filtered <- secchi.filtered %>%
-  left_join(unitConversion, by="units") %>%
-  mutate(convertedValue = value * conversion) %>%
-  filter(convertedValue < 150) 
-  
 group_by(secchi.filtered, area) %>% summarize(nSites = length(unique(wqx.id)))
```
Comment (jordansread) [inline R/createFigure4.R:242]: where does the unit conversion happen now?

Comment (ldecicco-USGS) [reply R/createFigure4.R:242]: https://github.com/USGS-R/geolakes/blob/master/R/createFigure4.R#L14

gets joined on line 33

Comment (eread-usgs) [reply R/createFigure4.R:242]: are the units converted to meters? If so I gave you the wrong number, @ldecicco-USGS, should be 150 ft or 46m

**Comment by ldecicco-USGS** [comment]
OK @jread-usgs you want to take this and incorporate your PIP + region stuff?

**Pull request merged**